library/errata_tool_cdn_repo.py

```diff
@@ -514,7 +516,76 @@ def ensure_packages_tags(client, name, check_mode, packages):
                                               desired_tags)
 
         changes.extend(package_changes)
-    return changes
+
+    # The value of current is useful to the caller so return
+    # that as well as the changes
```

Comment:
Thanks for explaining this change. Let's say why it's useful

```suggestion
    # The caller needs to know the list of changes and the current state in
    # order to support diff mode
```

Reply:
:+1: